app/js/lib/foxbox/recipes.js

```diff
@@ -6,10 +6,35 @@
 const p = Object.freeze({
   // Private properties.
   settings: Symbol('settings'),
-  net: Symbol('net')
+  net: Symbol('net'),
+
+  // Recipe private properties.
+  service: Symbol('service')
 });
 
-export default class Recipe {
+export class Recipe {
```

Comment:
Oh I haven't thought of that. I don't really want to expose it as raw data. Maybe we should leave it here then, hopefully that won't impact badly unit testing of these classes.